hudi-common/src/main/java/org/apache/hudi/common/model/HoodieAvroRecordMerger.java

```diff
@@ -39,7 +39,7 @@ public class HoodieAvroRecordMerger implements HoodieRecordMerger, OperationMode
 
   @Override
   public String getMergingStrategy() {
-    return HoodieRecordMerger.DEFAULT_MERGER_STRATEGY_UUID;
+    return HoodieRecordMerger.PAYLOAD_BASED_MERGER_STRATEGY_UUDID;
```

Comment:
We should add a validation that `HoodieAvroRecordMerger` should be used with this payload merging strategy only, if there is a change of decrepancy in the code, to be defensive.

Reply:
Is this done or unnecessary any more?